src/components/molecules/NavBar/NavBar.tsx

```diff
@@ -313,6 +313,17 @@ export const NavBar: React.FC<NavBarPropsType> = ({
                 <div>{navbarTitle}</div>
               )}
 
+              <div
+                className="nav-schedule nav-photobooth"
+                onClick={() =>
+                  openUrlInNewTab(
+                    "https://burn.sparklever.se/in/outsnappedphotoboothcamp"
+                  )
+                }
+              >
+                <p>Photobooth</p>
```

Comment:
This way, no need to target `p` if by any chance it needs to be replaced with `div` that has no styles in `global.scss` set

```suggestion
                <p className="NavBar__photoboot-title">Photobooth</p>
```

Reply:
fixed in cab515897d168494475f82a5a0970d9270d88094